.github/workflows/verify.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Build All Variants
-        run: mvn clean install -Pprod -Pjpa -Pinfinispan -Pkafka -Pstreams -Pasyncmem -pl !tests
+        run: mvn clean install -Pprod -Psql -Pinfinispan -Pstreams -Pasyncmem -pl !tests
```

Comment:
Unless we're removing definitely the Kafka storage in the `2.0.x` release I'm not in favour of removing it from the verify workflow.

Reply:
I think it's definitive, because the storage/kafka module is gone

Reply:
Sorry, haven't reviewed everything yet. If that's the case, then I fall back to my previous comment, to separate the Kafka storage stuff into a separate PR.